ra2ce/README.md

```diff
@@ -1,16 +1,29 @@
 # RA2CE
 
+## Ra2ce semantics
+
+- `ra2ce` is a [package](https://packaging.python.org/en/latest/tutorials/packaging-projects/),
+- everything else with an `__init__.py` is a subpackage (often written as (sub-)package).
+- single `.py` files are [modules](https://docs.python.org/3/tutorial/modules.html),
+- from a business perspective an `analysis` might be a module, but technically it can be implemented either as a (sub-package) or a module.
```

Comment:
```suggestion
- from a business perspective an `analysis` might be a module, but technically it can be implemented either as a (subpackage) or a module.
```